apps/status-api/src/modules/WhaleApiModule.ts

```diff
@@ -0,0 +1,61 @@
+import { Global, Injectable, Module } from '@nestjs/common'
+import { HealthIndicatorResult } from '@nestjs/terminus'
+import { WhaleApiClientProvider } from '../providers/WhaleApiClientProvider'
+import { ActuatorProbes, ProbeIndicator } from '@defichain-apps/libs/actuator'
+import { BlockchainInfo } from '@defichain/jellyfish-api-core/src/category/blockchain'
+
+@Injectable()
+export class WhaleApiProbeIndicator extends ProbeIndicator {
```

Comment:
We can also move the use the default RPC probe if we move to the other structure.